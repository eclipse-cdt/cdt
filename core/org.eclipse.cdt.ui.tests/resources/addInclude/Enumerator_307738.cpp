void test() {
  int i = Ns::ENUM_VALUE;
}
