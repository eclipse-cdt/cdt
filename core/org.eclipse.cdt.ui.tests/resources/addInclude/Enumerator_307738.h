namespace Ns {
  enum MyEnum {ENUM_VALUE};
}
