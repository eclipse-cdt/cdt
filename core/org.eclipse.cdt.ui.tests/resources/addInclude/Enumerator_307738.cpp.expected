#include "Enumerator_307738.h"

void test() {
  int i = Ns::ENUM_VALUE;
}
